.eslintrc

```diff
@@ -12,7 +12,8 @@
   "rules": {
     "max-len": "off", // Remove warnings on max line length exceeding 100 characters
     "react/require-default-props": "off", // Disabled the requirement to default all non-required props
-    "no-multi-assign": "off" // TODO: Decide if we want to turn this on. It was turned on in eslint-config-airbnb-base v14.1.0
+    "no-multi-assign": "off", // TODO: Decide if we want to turn this on. It was turned on in eslint-config-airbnb-base v14.1.0
+    "no-underscore-dangle": "off"
```

Comment:
added this to get around a linter error with accessing '_plugins' from the compiler.  Was not able to access them any other way without having another lint error occur.

Reply:
Instead of turning it off completely just allow '_plugins' to be accessed in the test file by adding this to the top of the file:
```/* eslint no-underscore-dangle: ["error", { "allow": ["_plugins"] }] */```